protokube/pkg/protokube/codecs.go

```diff
@@ -20,14 +20,15 @@ import (
 	"bytes"
 	"fmt"
 	"github.com/golang/glog"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/pkg/api/v1"
 	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/v1"
-	"k8s.io/kubernetes/pkg/runtime"
 
 	_ "k8s.io/kubernetes/pkg/api/install"
 )
 
 func encoder() runtime.Encoder {
+	// This encoder is kubernetes core, not kops encoder
 	yaml, ok := runtime.SerializerInfoForMediaType(api.Codecs.SupportedMediaTypes(), "application/yaml")
```

Comment:
I think I fixed it ... but @justinsb do we need the codec to parse kops api types as well?